### ISSUE TITLE
Support data_file_division of TSA for TSA GenBank files

### ISSUE THREAD

**Issue by kblin** [biopython/biopython#2447]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

```python
import sys; print(sys.version)
import platform; print(platform.python_implementation()); print(platform.platform())
import Bio; print(Bio.__version__)
```
```
3.7.4 (default, Jul  9 2019, 18:15:00)
[Clang 10.0.0 (clang-1000.11.45.5)]
CPython
Darwin-17.7.0-x86_64-i386-64bit
/Users/kblin/dev/libs/biopython/Bio/__init__.py:128: BiopythonWarning: You may be importing Biopython from inside the source tree. This is bad practice and might lead to downstream issues. In particular, you might encounter ImportErrors due to missing compiled C extensions. We recommend that you try running your code from outside the source tree. If you are outside the source tree then you have a setup.py file in an unexpected directory: /Users/kblin/dev/libs/biopython.
  format(_parent_dir), BiopythonWarning)
1.77.dev0
```

### Expected behaviour

First line of the GenBank file has "TSA" when writing TSA files.

### Actual behaviour

First lime of GenBank file has "UNK" when writing TSA file.

### Steps to reproduce

Run this in the Biopython basedir

```
#!/usr/bin/env python3
import io

from Bio import SeqIO


with open("Tests/GenBank/tsa_acropora.gb", "r") as infile:
    rec = SeqIO.read(infile, "genbank")
    infile.seek(0)
    first_line = infile.readline()

outfile = io.StringIO()
SeqIO.write([rec], outfile, "genbank")
outfile.seek(0)
first_line_written = outfile.readline()

assert first_line == first_line_written, "\n{}\nvs.\n{}".format(first_line, first_line_written)
```

**Comment by kblin** [comment]
I've got a patch for this.

**Comment by peterjc** [comment]
Closed by #2448

**Comment by peterjc** [comment]
Thank you - should merged this earlier, but we were distracted by Python 2 spring cleaning.
